lib/tic_tac_toe_web/models/game.rb

```diff
@@ -12,7 +14,7 @@ class Game
       CLASS_SIZE_4 = "size_four"
 
       def initialize(game, player_type, board_type)
-        @game = game
+        @game = GameUpdater.new(game).next_game_state
```

Comment:
This means we can't actually visualise the plays in a CvC game.
@game = next_state is a bit weird.